packages/components/src/DeliveryReclamation.purs

```diff
@@ -59,6 +61,7 @@ deliveryReclamation = make component { initialState, render, didMount }
 initialState :: State
 initialState =
   { publicationDate: Nothing
+  , doorCode: Just ""
```

Comment:
Why not `Nothing`?

Reply:
`Nothing` means that the form would fail validation until the field got some value. I want it to succeed with initial empty value.